python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -207,8 +214,12 @@ def extractStamps(self, inputExposure, refObjLoader=None):
         GMags = []
         ids = []
         wcs = inputExposure.getWcs()
-        # select stars within input exposure from refcat
-        withinCalexp = refObjLoader.loadPixelBox(inputExposure.getBBox(), wcs, filterName="phot_g_mean")
+        bmp = inputExposure.mask.getMaskPlaneDict()
+        # select stars within, or close enough to input exposure from refcat
+        inputExpBBox = inputExposure.getBBox()
+        dilatationExtent = geom.Extent2I(np.array(self.config.stampSize) - self.config.minPixelsWithinFrame)
+        withinCalexp = refObjLoader.loadPixelBox(inputExpBBox.dilatedBy(dilatationExtent), wcs,
+                                                 filterName="phot_g_mean")
```

Comment:
Is the filter always going to be phot_g_mean?

Reply:
For now, yes. The refcat used will always be Gaia, and the band always (Gaia) G.

This will only change with DM-25175, and I was planning on adjusting this when we get there. I added a TODO comment referencing the tickets for now.